string cmd
float32 duration